src/config/env.ts

```diff
@@ -1,39 +1,27 @@
-/* eslint-disable prefer-destructuring */
-import getConfig from 'next/config';
+export const SENTRY_DSN = process.env.NEXT_PUBLIC_SENTRY_DSN;
 
-const { publicRuntimeConfig } = getConfig();
-
-export const SENTRY_DSN = publicRuntimeConfig.NEXT_PUBLIC_SENTRY_DSN;
-export const SENTRY_ENV = publicRuntimeConfig.NEXT_PUBLIC_SENTRY_ENV;
-export const GRAASP_AUTH_HOST =
-  publicRuntimeConfig.NEXT_PUBLIC_GRAASP_AUTH_HOST;
-export const GRAASP_API_HOST = publicRuntimeConfig.NEXT_PUBLIC_API_HOST;
+export const SENTRY_ENV = process.env.NEXT_PUBLIC_SENTRY_ENV;
+export const GRAASP_AUTH_HOST = process.env.NEXT_PUBLIC_GRAASP_AUTH_HOST;
+export const GRAASP_API_HOST = process.env.NEXT_PUBLIC_API_HOST;
 export const GRAASP_PLAYER_HOST =
-  publicRuntimeConfig.NEXT_PUBLIC_GRAASP_PERFORM_HOST ||
-  'http://localhost:3112';
+  process.env.NEXT_PUBLIC_GRAASP_PERFORM_HOST || 'http://localhost:3112';
 export const GRAASP_BUILDER_HOST =
-  publicRuntimeConfig.NEXT_PUBLIC_GRAASP_BUILDER_HOST ||
-  'http://localhost:3111';
+  process.env.NEXT_PUBLIC_GRAASP_BUILDER_HOST || 'http://localhost:3111';
 export const GRAASP_ANALYTICS_HOST =
-  publicRuntimeConfig.NEXT_PUBLIC_GRAASP_ANALYTICS_HOST ||
-  'http://localhost:3113';
-export const NODE_ENV = publicRuntimeConfig.NODE_ENV;
+  process.env.NEXT_PUBLIC_GRAASP_ANALYTICS_HOST || 'http://localhost:3113';
 export const GRAASP_ACCOUNT_HOST =
-  publicRuntimeConfig.NEXT_PUBLIC_GRAASP_ACCOUNT_HOST ||
-  'http://localhost:3114';
-export const GA_MEASUREMENT_ID =
-  publicRuntimeConfig.NEXT_PUBLIC_GA_MEASUREMENT_ID;
-export const NEXT_PUBLIC_GRAASPER_ID =
-  publicRuntimeConfig.NEXT_PUBLIC_GRAASPER_ID;
-export const S3_FILES_HOST = publicRuntimeConfig.NEXT_PUBLIC_S3_FILES_HOST;
+  process.env.NEXT_PUBLIC_GRAASP_ACCOUNT_HOST || 'http://localhost:3114';
+// eslint-disable-next-line prefer-destructuring
+export const NODE_ENV = process.env.NODE_ENV;
+export const GA_MEASUREMENT_ID = process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID;
+export const GRAASPER_ID = process.env.NEXT_PUBLIC_GRAASPER_ID;
 export const SHOW_NOTIFICATIONS =
-  publicRuntimeConfig.NEXT_PUBLIC_SHOW_NOTIFICATIONS ||
+  process.env.NEXT_PUBLIC_SHOW_NOTIFICATIONS ||
```

Comment:
Careful here because if the value is `"false"` bad things will happen. I think we can remove the first part.